runestone/__main__.py

```diff
@@ -150,10 +150,10 @@ def build(all, wd):
     else:
         os.chdir(findProjectRoot())
     sys.path.insert(0, os.getcwd())
-    if not pathlib.Path(resource_filename("runestone", "dist/runestone.js")).exists():
-        click.echo("Error -- You are missing runestone.js please make sure")
-        click.echo("you have runestone installed correctly.")
-        click.echo("In a development environment run npm run build")
+    if not pathlib.Path(resource_filename("runestone", "dist/webpack.bundle.js")).exists():
+        click.echo("Error -- You are missing webpack.bundle.js.  Please make sure")
```

Comment:
Could we make this runestone.bundle.js - I am worried that we will run into name conflicts with a generic webpack.bundle.js name.  It will also make more sense for projects like PreTeXt to know that this is the runestone bundle.

Reply:
Also I assume that on the server side where we now include runestone.js we will change it to the bundle correct??

Reply:
Yes, plus we need to include a few more files -- everything listed in `runestone/dist/webpack_static_imports.json`.

I'm looking at docs on how to do the rename.

Reply:
Hmmm... `webpack_static_imports.json` is a file that gets generated by the `npm build` command.  What mechanism are we to use to get those loaded on the instructor interface pages??

Reply:
I think that's reasonably straightforward, since web2py has templates -- just read the JSON file for any of the books then transform this to `<script>` tags. Here's a crude sketch:

``` Python
{{ with open("path/to/webpack_static_imports.json", encoding="utf-8") as f:
    for js_import in json.loads(f.read()):
      response.write(f"<script src='_static/{js_import}'></script>\n")
}}
```